### ISSUE TITLE
Ask logged-in users to rate the app.

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-ios#535]
![image](https://user-images.githubusercontent.com/1689815/71544252-0a60fa80-297d-11ea-8e6b-edf5ecb50aa5.png)

**Comment by philippeauriach** [comment]
Having a bit of experience with this kind of features, I suggest the following : 
- having some kind of "decision tree" to ask only user actively using the app, or at least having used a specific feature (such as scanned a product or contributing), to avoid bad reviews from people not using the app deeply enough
- using directly the skstorereviewcontroller provided by apple : Apple decide if the popin is shown to the user (so we don't risk to show it more than 3 times a year), and it is embedded directly in the app, the user do not have to leave to the appstore. But this means we can not have our own dialog asking "are you enjoying the app" as, if the user clicks yes but then Apple does not show the rating stroe controller, it would be a bad experience (there is no way to know if the dialog was shown or not, Apple did this to not encourage ad-based apps that would reward the user for a good rate)

**Comment by teolemon** [comment]
Thanks for the input @philippeauriach 
I was thinking we could use the fact that the user is logged in as a proxy for happiness and engagement.

**Comment by anubhavsingh16** [comment]
@teolemon According to apple guideline we use only recommended API for rating iOS apps?

**Comment by anubhavsingh16** [comment]
@teolemon Can I do with Storekit framework?

**Comment by teolemon** [comment]
Yes, very much following the Apple Guidelines. That sound like the kind of feature where there are some ready made implementations available ?

**Comment by philippeauriach** [comment]
Yes the Storekit is pretty easy to use. I made a gist whih shows how to ask storekit only after the user compeltes a few events : https://gist.github.com/philippeauriach/311feff934205cff2f767b5e47653862 
(as the storekit implementation by Apple restricts the number of time it can be shown, we prefer to ensure that the user is actively using the app before asking him to rate it)
